e2e/pages/collections/svap-information/svap-information.readonly.spec.js

```diff
@@ -0,0 +1,93 @@
+import SvapInformationPage from './svap-information.po';
+
+let page;
+
+describe('the SVAP Information collection page', () => {
+  beforeEach(async () => {
+    page = new SvapInformationPage();
+    await page.open();
+  });
+
+  //below text will be updated, once PO confirmed exact text on svap information page
+  it('should have body text', async () => {
+    await expect(await page.getBodyText()).toContain('This list includes all 2015 Edition, including Cures Update, health IT products that have been INSERT TEXT HERE');
+    await expect(await page.getBodyText()).toContain('Please note that by default, only listings that are active or suspended are shown in the search results.');
+  });
+
+  it('should have table headers in a defined order', async () => {
+    const expectedHeaders = ['CHPL ID', 'Certification Edition', 'Developer', 'Product', 'Version', 'Status', 'SVAP Information', 'SVAP Notice URL', 'Actions'];
```

Comment:
```suggestion
    const expectedHeaders = ['CHPL ID', 'Certification Edition', 'Developer', 'Product', 'Version', 'Status', 'SVAP Information', 'SVAP Notice', 'Actions'];
```

Changed the header after talking with the PO. This change should match the updates